init/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "init-rs"
```

Comment:
For the crate name I think we should avoid using the `-rs` extension, since it's implied. By contrast, the repo name including -rs is a good way to distinguish from other language impls.

Reply:
Also, not sure if "init" on its own is the right name for the crate, could be ambiguous. I'd been thinking something like pid1-exe, but not feeling great about that either.

Reply:
The reason I used `init-rs`, is because there is already an `init` crate at `crates.io`.  Yeah, probably `pid1-exe` is better: But do note that I have configured the binary name as `pid1`. Do you think that's okay ?

Reply:
Yeah, I think that's a reasonable setup.

Reply:
Cool, I have renamed it to `pid1-exe`.